app/code/community/Reverb/ReverbSync/etc/config.xml

```diff
@@ -333,6 +333,12 @@
         <order_sync_reverb_source_url>/api/my/orders/selling/all?created_start_date=%s</order_sync_reverb_source_url>
         <!-- Default orders to be synced to the Admin store -->
         <store_to_sync_order_to>0</store_to_sync_order_to>
+        <!-- The set of order statuses which should be regarded as constituting NOT being a paid order -->
+        <non_paid_order_statuses>
```

Comment:
In this case, if the only order update which comes through for order X is "shipped", then no order will be created in the Magento system for order X. This will affect the reporting statistics in the Magento system, and will prevent customer service personnel from being able to reference that order in the Magento admin panel. Is this desired?